test/test_podman_baseline.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+# test_podman_baseline.sh
+# A script to be run at the command line with Podman installed.
+# This should be run against a new kit to provide base level testing
+# on a freshly installed machine with no images or container in
+# play.  This currently needs to be run as root.
+#
+# To run this command:
+#
+# /bin/bash -v test_podman_baseline.sh
+
+########
```

Comment:
Maybe a `set -eu` somewhere in here to make sure we exit on errors?

Reply:
I think Tom prefers running all tests and then examining the output for multiple failures.  Adding set -eu would cause the script to fail right away.  If there was a simple way to increment a ctr of failed commands so it was easy at the end to tell if the script discovered some unexpected failures, that might help out a human reviewing the session.  @ipbabble @TomSweeneyRedHat What do you think?